core/src/components/input/input.scss

```diff
@@ -291,9 +291,19 @@
 
   transition: background-color 15ms linear;
 
+  border-bottom: var(--border-width) var(--border-style) var(--border-color);
```

Comment:
Should a shaped input still have the bottom border radius even with no fill? It looks a little odd:
![shaped no fill](https://i.gyazo.com/afc66c6b095ed96c184ffacb5a601150.png)

Reply:
That works for me 👍